src/utils/database.ts

```diff
@@ -0,0 +1,37 @@
+const { existsSync, readFileSync, writeFileSync, unlinkSync } = require('fs');
+
+const defaultFilePath = 'saves/data.json';
+
+module.exports = {
+  hasGameFile(): boolean {
+    if (existsSync(defaultFilePath)) return true;
+    return false;
+  },
+
+  load(): unknown {
```

Comment:
IMO this one can just throw if it fails and kill the process. If the calling code doesn't want to throw it should check if there's a gamefile first

Reply:
Returns `null` on error

Reply:
We decided to return null